CHANGES/574.bugfix

```diff
@@ -0,0 +1,3 @@
+Correctly identified base_path as required when creating a Distribution.
+
+(affected rpm, file, and python distribution create)
```

Comment:
Not sure if the empty line collides with the markdown formatting. Can you please check on this?

Reply:
The result of `towncrier build --draft` is:
```
...
- Fixed ``KeyError: 'missing_field'`` error when required fields are not supplied.
  [#572](https://github.com/pulp/pulp-cli/issues/572)
- Correctly identified base_path as required when creating a Distribution.

  (affected rpm, file, and python distribution create)
  [#574](https://github.com/pulp/pulp-cli/issues/574)
```

Reply:
Does that render properly in Markdown is my question. XD
(It's either MD or RST where it does not... cannot remember)

Reply:
I'm not sure what you mean by "breaks", here. The towncrier process works, the resulting changelog is readable, and we have done this in other changelog entries in the past (altho poss not in pulp-cli). What are you expecting here, that you're not seeing?

> Would you mind getting container and ansible distributions in line too?

container and ansible already have it defined as required=True at create-time.

Reply:
> I'm not sure what you mean by "breaks", here. The towncrier process works, the resulting changelog is readable, and we have done this in other changelog entries in the past (altho poss not in pulp-cli). What are you expecting here, that you're not seeing?

I remember that when releasing the the changelog was not rendered properly in html. And i'm not sure it was here or in one of the rst based projects. But I just realized i cannot put the onus on you to inverstigate this further.
> 
> > Would you mind getting container and ansible distributions in line too?
> 
> container and ansible already have it defined as required=True at create-time.

Yes, what i thought was making their implementation similar to the ones you just changed.